openlibrary/plugins/openlibrary/js/index.js

```diff
@@ -123,6 +123,37 @@ jQuery(function () {
             .then(module => module.initEditionsTable());
     }
 
+
+    /* js code for the Share module in the book page
+     * Coding By CodingNepal - youtube.com/codingnepal */
+    const viewBtn = document.querySelector('.view-modal'),
+        popup = document.querySelector('.popup'),
+        close = popup.querySelector('.close'),
+        field = popup.querySelector('.copy-field'),
+        input = field.querySelector('input'),
+        copy = field.querySelector('button');
+
+    viewBtn.onclick = ()=>{
+        popup.classList.toggle('show');
+    }
+    close.onclick = ()=>{
+        viewBtn.click();
+    }
+
+    copy.onclick = ()=>{
+        input.select(); //select input value
+        if (document.execCommand('copy')) { //if the selected text copy
+            field.classList.add('active');
+            copy.innerText = 'Copied';
+            setTimeout(()=>{
+                window.getSelection().removeAllRanges(); //remove selection from document
+                field.classList.remove('active');
+                copy.innerText = 'Copy';
+            }, 3000);
+        }
+    }
+
+
```

Comment:
Much of this will not be relevant when you switch to Colorbox.  Whatever JS code is needed to display the Colorbox modal should be moved to a separate file (like `Share.js`), and imported here if share links exist on the page.